app/assets/javascripts/discourse/app/models/category.js

```diff
@@ -200,7 +192,8 @@ const Category = RestModel.extend({
     const url = id ? `/categories/${id}` : "/categories";
 
     return ajax(url, {
-      data: {
+      contentType: "application/json",
+      data: JSON.stringify({
```

Comment:
It didn't default to "application/json" before? 👀

Reply:
No, our default is form-data. I had to change this call because the new data structure (`required_tag_groups: [{...}, {...}]`) cannot be represented consistently in form-data.